### ISSUE TITLE
A feature to hide the trash button

### ISSUE THREAD

**Issue by CLEMENT2OOO** [mozilla-mobile/focus-android#5269]
Here is my little problem. Since I don't use the pink trash button that appears every time you browse a web page, I'd like a feature in the application settings that would allow it to be hidden.

Thanks

**Comment by Poopooracoocoo** [comment]
Focus was designed to be an ephemeral browser though. It defeats the entire point. I personally find it rather easy to accidentally hit though and an option to move it to the toolbar would be nice.

---
Unrelated but this is my first comment in a long time in this repo so I'll say it anyways, especially since Focus has been getting some attention from the devs:
- I'd like some improvements to the toolbar behaviour to match chrome or even the old Focus behaviour.
- I'd like to be able to keep my browsing data (see DDG Browser or Safari's nice implementation of private browsing). Focus easily loses browsing sessions as soon as it goes out of memory, and like Firefox, it's not very performant and so that happens often. 
- I want to be able to open links in another app from the link context menu. Focus recently lost this.
- And this is a bug: I want my tabs to keep their browsing history and current page. 
- I want ads to be blocked like they used to be (at the launch of Focus)

**Comment by amedyne** [comment]
@channingcarter @brampitoyo What do you think of this?

**Comment by Poopooracoocoo** [comment]
@amedyne I don't even know who to mention to get the attention of someone who'll fix the issues I mentioned above. :'(

**Comment by amedyne** [comment]
@Poopooracoocoo We have a bug that we will use to track the improvements for the toolbar behavior: https://github.com/mozilla-mobile/focus-android/issues/5452

For all the other items you mentioned, there are GH issues that are open. I created a new issue report for the Focus phone battery use.

**Comment by brampitoyo** [comment]
I agree with @Poopooracoocoo that Focus is designed as an easy-to-erase browser, so it must be easy to erase!

I’d be open on discussing moving the Floating Action Button – to the toolbar perhaps? But then it would be easy to hit that if you meant to go to the menu. This isn’t an easy problem to solve.
